src/storage/StorageServiceHandler.h

```diff
@@ -74,6 +85,14 @@ class StorageServiceHandler final : public cpp2::StorageServiceSvIf {
 private:
     kvstore::KVStore* kvstore_ = nullptr;
     meta::SchemaManager* schemaMan_;
+    StorageStats getOutBoundQpsStat_;
+    StorageStats getInBoundQpsStat_;
+    StorageStats outBoundStatsQpsStat_;
+    StorageStats inBoundStatsQpsStat_;
+    StorageStats vertexPropsQpsStat_;
+    StorageStats edgePropsQpsStat_;
+    StorageStats addVerticesQpsStat_;
+    StorageStats addEdgesQpsStat_;
```

Comment:
maybe should move all of the property of status to each class which use it, It is not a property of StorageServiceHandler, and just used them to construct sub processor, and it is a property of each sub processor more suitable

Reply:
The reason is partly because of each time we create a new instance of processor to handle a request. And we have to maintain the same stat id for all of them.